toolbox/fdc3-workbench/src/components/common/AccordionContent.tsx

```diff
@@ -66,6 +74,9 @@ export const AccordionContent: React.FC<AccordionContentProps> = ({ title, child
 			>
 				<Typography variant="h5" className={classes.accordionTitle}>
 					{title}
+					{icon && <Tooltip title={icon} aria-label={icon}>
+						<InfoIcon className={classes.icon}/>
+					</Tooltip>}
```

Comment:
just curious what this does in tsx, is it an an interpolated string value with a tooltip applied to it? In other places I saw that the tooltip element was wrapping its target

Reply:
The tooltip here wraps the InfoIcon target similar to other instances of the tooltip.